sips/sip-013/sip-013-pox-upgrade.md

```diff
@@ -0,0 +1,373 @@
+the stacker has not delegated to an operator, that their funds are
+currently stacked, and that they have enough unlocked funds to cover
+the increase. The caller may *not* supply a new reward address for the
+increase.
+
+The special case handler for the PoX contract in the Clarity VM will
+check this method's return value and set the locked amount in the
+stacker's STX account to correspond to the increased amount.
+
+### 4. `delegate-stack-increase`
+
+This method allows operators to lock additional funds for one of their
+delegation participants for the remaining cycles on that user's
+current lock.
+
+This method checks that the delegator is still authorized on behalf of
+the given stacker (and remains authorized until the end of the
+locked-for period), that the increased amount remains less than the
+delegation's `amount-ustx` field, that the user is currently locked,
+and that the user has enough unlocked funds to cover the increase.
+
+Note that, just as with the existing `delegate-stack-stx` function,
+this method locks *but does not commit* the user's STX. The delegation
+operator must invoke `stack-aggregation-commit` to set a reward address
+for the newly locked funds.
+
+The special case handler for the PoX contract in the Clarity VM will
+check this method's return value and set the locked amount in the
+stacker's STX account to correspond to the increased amount.
+
+### 5. `stack-unlock`
+
+This method allows direct stackers (not delegation participants) to
+unlock their funds *after* the active reward cycle: removing the
+account from all future reward sets.
+
+The special case handler for the PoX contract in the Clarity VM will
+check this method's return value and set the stacker's STX account to
+"auto-unlock" at the start of the next reward cycle (i.e., the user
+account will behave as if the current cycle is their last active
+cycle).
+
+### 6. `stack-unlock-early`
+
+This method allows direct stackers (not delegation participants) to unlock
+their funds during the *current* reward cycle if, and only if, the user's
+stacked amount was not sufficient to obtain a rewards slot.
+
+This method will behave similarly to `stack-unlock`, removing the account
+from all future reward sets. The special case handler for the PoX contract
+in the Clarity VM will check this method's return value and set the stacker's
+STX account to "auto-unlock" at the _next_ Stacks block height.
+
+### 7. `delegator-stack-unlock`
+
+This method allows a user who delegated their PoX functionality
+to an operator to unlock a locked _but not committed_ amount from their
+account. The reason that the amount cannot have been committed yet is that
+the operator and the PoX contract cannot distinguish between which accounts
+have been used during a given cycle's commitment-- and allowing users to
+"uncommit" from the operator's pool would introduce denial-of-service vectors
+to the pool.
+
+### 8. `stack-aggregation-rollback`
```

Comment:
Under what circumstances would an operator use this method?

Reply:
If they wanted to increase the amount that they committed, but the increase by itself wouldn't be enough to get another entry in the rewards table.